tests/api2/test_audit_basic.py

```diff
@@ -35,147 +84,156 @@ def initialize_for_smb_tests(request):
-        'services': ['SMB'],
-        'query-filters': [['username', '=', SMBUSER]],
-        'query-options': {'count': True}
-    })
-    assert ops > 0
-
-
-def test_audit_export(request):
-    depends(request, ["AUDIT_OPS_PERFORMED"], scope="session")
-    for backend in ['CSV', 'JSON', 'YAML']:
-        report_path = call('audit.export', {'export_format': backend}, job=True)
-        assert report_path.startswith('/audit/reports/root/')
-        st = call('filesystem.stat', report_path)
-        assert st['size'] != 0, str(st)
-
-        job_id, path = call("core.download", "audit.download_report", [{
-            "report_name": os.path.basename(report_path)
-        }], f"report.{backend.lower()}")
-        r = requests.get(f"{url()}{path}")
-        r.raise_for_status()
-        assert len(r.content) == st['size']
-
-
-def test_audit_export_nonroot(request):
-    depends(request, ["AUDIT_OPS_PERFORMED"], scope="session")
-
-    with unprivileged_user_client(roles=['SYSTEM_AUDIT_READ', 'FILESYSTEM_ATTRS_READ']) as c:
-        me = c.call('auth.me')
-        username = me['pw_name']
-
+# =====================================================================
+#                           Tests
+# =====================================================================
+class TestAuditConfig:
+    def test_audit_config_defaults(self, init_audit):
+        (config, dataset) = init_audit
+
+        # Confirm existence of config entries
+        for key in [k for k in AUDIT_DATASET_CONFIG]:
+            assert key in config, str(config)
+
+        # Confirm audit default config settings
+        assert config['retention'] == AUDIT_CONFIG.defaults['retention']
+        assert config['quota'] == AUDIT_CONFIG.defaults['quota']
+        assert config['reservation'] == AUDIT_CONFIG.defaults['reservation']
+        assert config['quota_fill_warning'] == AUDIT_CONFIG.defaults['quota_fill_warning']
+        assert config['quota_fill_critical'] == AUDIT_CONFIG.defaults['quota_fill_critical']
+        assert config['remote_logging_enabled'] is False
+        for key in ['used', 'used_by_snapshots', 'used_by_dataset', 'used_by_reservation', 'available']:
+            assert key in config['space'], str(config['space'])
+
+        for service in ['MIDDLEWARE', 'SMB', 'SUDO']:
+            assert service in config['enabled_services']
+
+        # Confirm audit dataset settings
+        for key in [k for k in AUDIT_DATASET_CONFIG if AUDIT_DATASET_CONFIG[k] == 'zfs']:
+            assert get_zfs('zfs', key, dataset) == config[key], f"config[{key}] = {config[key]}"
+
+    def test_audit_config_dataset_defaults(self):
+        """ Confirm Audit dataset uses Audit default settings """
+        ds_config = call('audit.get_audit_dataset')
+        assert ds_config['org.freenas:quota_warning'] == AUDIT_CONFIG.defaults['quota_fill_warning']
+        assert ds_config['org.freenas:quota_critical'] == AUDIT_CONFIG.defaults['quota_fill_critical']
```

Comment:
Huh.  I see 4 entries ...
- `org.freenas:quota_warning`
- `org.freenas:quota_critical`
- `org.freenas:refquota_warning`
- `org.freenas:refquota_critical`
From the PR description you mention `refquota`, so shouldn't we be testing that (instead?)